tests/testTriggers.go

```diff
@@ -3698,6 +3702,96 @@ func TriggerAutoFsTrim(contexts *[]*scheduler.Context, recordChan *chan *EventRe
 
 }
 
+// TriggerVolumeUpdate enables to test volume update
+func TriggerVolumeUpdate(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: UpdateVolume,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	context("Validate update of the volumes", func() {
+
+		Step("Update Io priority on volumes ",
+			func() {
+				updateIOPriorityOnVolumes(contexts, event)
+				logrus.Info("Update IO priority call completed")
+			})
+	})
+}
+
+//This method is responsible for updating IO priority on Volumes if volume is labelled with
+//IO Priority. If lebelled with High and volume is updated to HIGH, next iteration it will reset
+//back to low.
+func updateIOPriorityOnVolumes(contexts *[]*scheduler.Context, event *EventRecord) {
+	for _, ctx := range *contexts {
+		var appVolumes []*volume.Volume
+		var err error
+		appVolumes, err = Inst().S.GetVolumes(ctx)
+		UpdateOutcome(event, err)
+		if len(appVolumes) == 0 {
+			UpdateOutcome(event, fmt.Errorf("found no volumes for app "))
+		}
+		for _, v := range appVolumes {
+			isPureVol, err := Inst().V.IsPureVolume(v)
+			if err != nil {
+				UpdateOutcome(event, err)
+			}
+			if isPureVol {
+				logrus.Warningf(
+					"Autofs Trim is not supported for Pure DA volume: [%s]",
+					"Skipping autofs trim status on pure volumes", v.Name,
+				)
+				continue
+			}
+			logrus.Infof("Getting info from volume: %s", v.ID)
+			appVol, err := Inst().V.InspectVolume(v.ID)
+			if err != nil {
+				logrus.Errorf("Error inspecting volume: %v", err)
+			}
+			if requiredPriority, ok := appVol.Spec.VolumeLabels["priority_io"]; ok {
+				if !setIoPriority {
```

Comment:
what is the use case when setIoPriority is true?

Reply:
Volumes which expects IO priority to be set will be labelled with IO priority type (I have observed that volume are either labelled with low , high or no label.)
I set  setIoPriority  to True, meaning I need to set the volume  with IO priority according to the label specified with PVC.
[ if high, priority should be set to "High" , if "low" priority should be set to "LOW"]
After this is done , I set setIoPriority to false so that I can reset the priority on the volumes during next time when event is triggered [ This is to make sure that next iteration can update the priority back]

Reply:
Thanks @lsrinivas-pure for checking build status. I made required changes to keep comments in go format
The build is passing https://app.travis-ci.com/github/portworx/torpedo/builds/253272464
Could you please approve if there is no other comments